cookbooks/rubygems-app/recipes/logrotate.rb

```diff
@@ -11,6 +11,7 @@
   options %w(missingok nocreate compress delaycompress dateext notifempty sharedscripts)
   postrotate [
     '    [ -f /etc/service/unicorn/supervise/pid ] && kill -USR1 `cat /etc/service/unicorn/supervise/pid`',
-    '    [ -f /etc/service/delayed_job/supervise/pid ] && kill -USR1 `cat /etc/service/delayed_job/supervise/pid`'
+    '    [ -f /etc/service/delayed_job/supervise/pid ] && kill -USR1 `cat /etc/service/delayed_job/supervise/pid`',
+    '    [ -f /etc/service/shoryuken/supervise/pid ] && kill -USR1 `cat /etc/service/shoryuken/supervise/pid`'
```

Comment:
@dwradcliffe Glad you asked!

`USR1` is actually shoryuken's graceful shutdown signal. It doesn't have a "reopen logs" signal like unicorn, nginx, etc. So this works works in the sense that shoryuken will restart, hence opening new logs. But it's not obvious. I'll update to just do a more obvious `/etc/init.d/shoryuken restart`.

See https://github.com/phstc/shoryuken/wiki/Signals for more info.